pkg/virt-controller/watch/vmi.go

```diff
@@ -147,6 +147,7 @@ func NewVMIController(templateService services.TemplateService,
 	recorder record.EventRecorder,
 	clientset kubecli.KubevirtClient,
 	dataVolumeInformer cache.SharedIndexInformer,
+	storageProfileInformer cache.SharedIndexInformer,
```

Comment:
we should include this here:  https://github.com/kubevirt/kubevirt/blob/916397f0003979783e69b1428d061d07ee6b1bb0/pkg/virt-controller/watch/vmi.go#L177-L179

Reply:
Nice catch!